.gitignore

```diff
@@ -10,7 +10,12 @@ _build/
 _build/*
 book/_build/*
 _notebook_cache/*
-notebooks/Source.gv
-notebooks/*.pkl
-notebooks/*.csv*
+book/notebooks/Source.gv
+book/notebooks/*.pkl
+book/notebooks/*.csv*
 dataset*
+src/notebooks/*.nc
+src/notebooks/*.npy
+src/notebooks/*.gif
```

Comment:
Can we generally exclude *.gif and the other extensions here? that might help others in the future to not commit anything uneccessary

Reply:
We cannot generally exclude *.gif because some of them are being used in the notebooks.

Reply:
Ok thanks for clarifing that.